Tabnine/build.gradle

```diff
@@ -101,7 +101,7 @@ tasks.getByName("runIde").configure {
 
 tasks {
     compileKotlin {
-        dependsOn(ktlintFormat)
+//        dependsOn(ktlintFormat)
```

Comment:
show me how to actually run the lint and not just complain about it. 

It doesn't show any errors , just complains this useless thinig